### PR TITLE
Add gray translucent colors to theme

### PR DESCRIPTION
**Pull request by robinandeer**
## Describe your changes

Add translucent colors to the theme

Add more text colors

Add semantic gray colors based on translucent values

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by robinandeer** [comment]
Current dependencies on/for this PR:
* main
  * **PR #1434** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1434" target="_blank">Graphite</a> 
    * **PR #1435** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1435" target="_blank">Graphite</a> 
      * **PR #1437** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1437" target="_blank">Graphite</a> 
        * **PR #1440** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1440" target="_blank">Graphite</a> 
          * **PR #1441** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1441" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1441?utm_source=stack-comment).

**Review by alebedev — APPROVED** [review]
Looks good

File: packages/ui/src/lib/theme/colors/colors.ts [hunk 1]
```diff
@@ -13,6 +13,21 @@ export const gray = {
   1000: 'hsl(0, 0%, 7%)',
 } as const
 
+export const grayTranslucent = {
+  offWhite: 'hsla(0, 0%, 98%, 0.6)',
```
Comment (gustaveen) [inline packages/ui/src/lib/theme/colors/colors.ts:17]: nit: I think we can use 25 for this, the same as the gray scale and have an alias maybe? but I'm good with either 👍

**Review by gustaveen — APPROVED** [review]

File: packages/ui/src/lib/theme/colors/colors.ts (hunk 1 above)
Comment (robinandeer) [reply packages/ui/src/lib/theme/colors/colors.ts:17]: right, just wanted to stay as close to the figma file as possible

**Comment by robinandeer** [comment]
⏳ This pull request is set to merge as part of a Graphite merge job
Stack job ID: awvOMljvedZZqM3RSp8w.
See details on [graphite.dev](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1441)

**Pull request merged**